test/session.js

```diff
@@ -15,7 +15,7 @@ var Cookie = require('../session/cookie')
 var min = 60 * 1000;
 
 describe('session()', function(){
-  it('should export constructors', function(){
+  it('should export varructors', function(){
```

Comment:
Find and replace error here possibly?  The test should be left as: "should export constructors".

Reply:
haha, best s// error ever